kaldb/src/main/java/com/slack/kaldb/zipkinApi/Span.java

```diff
@@ -0,0 +1,91 @@
+package com.slack.kaldb.zipkinApi;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.HashMap;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Span {
+
+  private final String id;
+  private final String traceId;
+  private final String parentId;
+  private final String name;
+  private final long timestamp;
+  private final long duration;
+  private final String kind;
+  private final HashMap<String, Object> localEndpoint;
+  private final HashMap<String, Object> remoteEndpoint;
+
+  private final List<HashMap<String, Object>> annotations;
+  private final HashMap<String, String> tags;
+
+  public Span(
```

Comment:
Same with this class. Can we just leave  a comment here for now that this class was for initial testing ( it's currently used by the main method and not any actual code ) and we will remove this class eventually.